CodeConverter/CSharp/CommonConversions.cs

```diff
@@ -428,15 +473,15 @@ private IEnumerable<CSSyntaxKind> CsSyntaxAccessibilityKind(Accessibility declar
         {
             switch (declaredAccessibility) {
                 case Accessibility.Private:
-                    return new[] { CSSyntaxKind.PrivateKeyword };
+                    return new[] {CSSyntaxKind.PrivateKeyword};
                 case Accessibility.Protected:
-                    return new[] { CSSyntaxKind.ProtectedKeyword };
+                    return new[] {CSSyntaxKind.ProtectedKeyword};
                 case Accessibility.Internal:
-                    return new[] { CSSyntaxKind.InternalKeyword };
+                    return new[] {CSSyntaxKind.InternalKeyword};
                 case Accessibility.ProtectedOrInternal:
-                    return new[] { CSSyntaxKind.ProtectedKeyword, CSSyntaxKind.InternalKeyword };
+                    return new[] {CSSyntaxKind.ProtectedKeyword, CSSyntaxKind.InternalKeyword};
                 case Accessibility.Public:
-                    return new[] { CSSyntaxKind.PublicKeyword };
+                    return new[] {CSSyntaxKind.PublicKeyword};
```

Comment:
I realize the repo is pretty much half and half on this convention. If you want to standardize it, please create a separate PR for that

Reply:
I also reverted this with the other formatting changes in this file (might as well).